Apigee/SmartDocs/Revision.php

```diff
@@ -343,7 +351,7 @@ public function load($revisionId = null)
         if (is_int($revisionId) && $revisionId < 1) {
             throw new ParameterException('Cannot load a revision number less than 1.');
         }
-        $this->get($revisionId . '?expand=true');
+        $this->get(rawurlencode($this->apiId) . '/revisions/' . $revisionId . '?expand=true');
```

Comment:
I agree.  I modified all of these calls to deal with the tension between the existing Revision.load() and the new endpoint that I have been given:
https://:host/v1/organizations/:org/apimodels/:model/import/file?format=swagger

The Revision load uses the modelId instead of the modelName in the endpoint it uses.  Is this a backend issue?